danger-kotlin-library/src/main/kotlin/systems/danger/kotlin/tools/shell/ShellExecutor.kt

```diff
@@ -9,7 +9,7 @@ interface ShellExecutor {
     fun execute(command: String, arguments: List<String> = emptyList()): String
 }
 
-class ShellExecutorImpl : ShellExecutor {
+internal class ShellExecutorImpl : ShellExecutor {
```

Comment:
our internal implementation for the shell executor shouldn't be visible from the danger file, we can access by interface